mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -67,11 +69,16 @@ CLASS zr net/minecraft/util/math/MathHelper
 		ARG 0 array
 	METHOD b fastFloor (D)I
 	METHOD b floorMod (DD)D
-	METHOD b lerpClamped (DDD)D
+	METHOD b clampedLerp (DDD)D
```

Comment:
Both names make sense. Any reason for this rename?

Reply:
Shouldn't we be using prefixes everywhere?

Reply:
This is called by `clampedLerp` in `RealmsMath`.

Moreover, the `average([D)D` method is called by `average` in `RealmsMath`, but I think `mean` may be a better name. Should I change that?

Reply:
`mean` is good

Reply:
meanAverage and clamedLerp, IMO

Reply:
Aren't mean and average synonyms? That would be like saying "averageAverage"...

Reply:
Will leave it `average`.
![image](https://user-images.githubusercontent.com/7806504/55682168-79289000-58e4-11e9-82c5-d8494c234d2c.png)